src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
@srsiva There will be no breaking changes in the next version of PowerShell.  You must mitigate any breaks in the PowerShell API surface with code, or you will not be able to merge this library into PowerShell.  There are several strategies for mtigating breaking changes.  Please contact ius if you have questions abotu this.  Otherwise ack and this is OK from our perspective.